runtime/frequency/src/lib.rs

```diff
@@ -459,7 +459,7 @@ impl pallet_democracy::Config for Runtime {
 	/// vote immediately and asynchronously in a similar manner to the emergency origin.
 	/// Requires TechnicalCommittee to be unanimous.
 	type InstantOrigin = EitherOfDiverse<
-		pallet_collective::EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 1>,
+		pallet_collective::EnsureProportionAtLeast<AccountId, TechnicalCommitteeInstance, 1, 1>,
```

Comment:
Can we name them to something "collective" which would be the reason it got missed from the following story 
TechnicalCouncilCollective

https://github.com/LibertyDSNP/frequency/issues/350

Probably believe the confusion between Instance and Collective naming

Reply:
Yes, I will rename to make them consistent with each other.